### PR TITLE
Add raw_member_remove event

### PR DESCRIPTION
**Pull request by Gugu7264**
## Summary

Adds a raw_member_remove event, useful in case the member isn't cached (or chunking is disabled) which makes the client not fire an event at all when a member leaves a guild.

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by ooliver1 — CHANGES_REQUESTED** [review]
did you add anything to the docs?

File: nextcord/raw_models.py [hunk 1]
```diff
@@ -52,6 +54,7 @@
     'RawReactionClearEmojiEvent',
     'RawIntegrationDeleteEvent',
     'RawTypingEvent',
+    "RawMemberRemoveEvent"
```
Comment (vcokltfre) [inline nextcord/raw_models.py:57]:
```suggestion
    "RawMemberRemoveEvent",
```

File: nextcord/raw_models.py [hunk 2]
```diff
@@ -311,4 +314,21 @@ def __init__(self, data: TypingEvent) -> None:
...
+
+class RawMemberRemoveEvent(_RawReprMixin):
+    """Represents the payload for a :func:`on_raw_member_remove` event.
+    .. versionadded:: 2.0
+    Attributes
+    -----------
+    guild_id: :class:`int`
+        The guild ID where the member left from.
+    user: :class:`User`
+        The user who left the guild."""
```
Comment (vcokltfre) [inline nextcord/raw_models.py:328]:
```suggestion
        The user who left the guild.
    """
```

**Review by vcokltfre — CHANGES_REQUESTED** [review]
Looks good for the most part, just a couple of nitpicks I'd like fixed, the trailing commas are good practice for the future, since other things added won't require 2 lines of diff, and the docstring change is to remain consistent with other docstrings

File: nextcord/raw_models.py (hunk 2 above)
Comment (ooliver1) [reply nextcord/raw_models.py:328]: ^ for this, https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation is the reason

**Review by TAG-Epic — APPROVED** [review]
Not tested

**Review by ooliver1 — APPROVED** [review]

**Comment by ooliver1** [comment]
3 approves so merging (commits after vco were his requested changes)

**Pull request merged**